src/main/java/uk/gov/hmcts/reform/sscs/service/SaveCorrespondenceAsyncService.java

```diff
@@ -25,9 +27,24 @@ public SaveCorrespondenceAsyncService(CcdNotificationsPdfService ccdNotification
         this.ccdNotificationsPdfService = ccdNotificationsPdfService;
     }
 
+    @Autowired
+    private LetterAsyncConfigProperties letterAsyncConfigProperties;
+
     @Async
-    @Retryable(maxAttemptsExpression = "#{${letterAsync.maxAttempts}}", backoff = @Backoff(delayExpression = "#{${letterAsync.delay}}", multiplierExpression = "#{${letterAsync.multiplier}}", random = true))
+    @Retryable(maxAttemptsExpression = "#{@letterAsyncConfigProperties.maxAttempts}", backoff = @Backoff(delayExpression = "#{@letterAsyncConfigProperties.delay}", multiplierExpression = "#{@letterAsyncConfigProperties.multiplier}", random = true, maxDelayExpression = "#{@letterAsyncConfigProperties.maxDelay}"))
```

Comment:
Sonar was complaining about this format so had to change it. https://sonarsource.atlassian.net/browse/SONARJAVA-4917